x/yarpctest/request_unary.go

```diff
@@ -58,11 +58,25 @@ func HTTPRequest(options ...api.RequestOption) api.Action {
 		require.NoError(t, out.Start())
 		defer func() { assert.NoError(t, out.Stop()) }()
 
-		resp, cancel, err := sendRequest(out, opts.GiveRequest, opts.GiveTimeout)
-		defer cancel()
-		validateError(t, err, opts.WantError)
-		if opts.WantError == nil {
-			validateResponse(t, resp, opts.WantResponse)
+		for i := 0; i < opts.RetryCount+1; i++ {
```

Comment:
We can avoid the +1 by reframing the value as an "AttemptCount", which would include the first.

Reply:
done